Easydict/Feature/Configuration/NewConfiguration.swift

```diff
@@ -0,0 +1,595 @@
+//
+//  NewConfiguration.swift
+//  Easydict
+//
+//  Created by ljk on 2024/1/2.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Defaults
+import Foundation
+
+class Configuration: NSObject {
+    private(set) static var shared = Configuration()
+
+    var appDelegate = NSApp.delegate as? AppDelegate
+
+    var updater: SPUUpdater? {
+        appDelegate?.updaterController.updater
+    }
+
+    @Default(.firstLanguage)
+    var firstLanguage: Language
+
+    @Default(.secondLanguage)
+    var secondLanguage: Language
+
+    @Default(.queryFromLanguage)
+    var from: Language
```

Comment:
It would be better to rename `from` and `to` to a more readable name like `firstLanguage` above.